main/plugins/org.talend.hadoop.distribution/src/main/java/messages.properties

```
@@ -42,7 +42,7 @@ ESparkMode.SPARK_LOCAL=Local
 ESparkMode.CDE=Cloudera Data Engineering
 ESparkMode.DATAPROC=Dataproc
 ESparkMode.SYNAPSE=Synapse
-ESparkMode.HDI=HD Insight
+ESparkMode.HDInsight=HDInsight

```

```suggestion
ESparkMode.HDInsight=HDInsight
ESparkMode.HDINSIGHT=HDInsight
```
If you really want to change from HDI on key part we should stay with capital letters to be consistent